### DISCUSSION TITLE
Validation error File upload error

### DISCUSSION THREAD

**Discussion by R1deNn** [orchidsoftware/platform#2849] [Q&A] [answered]
Hello. When I try to upload an image or file, I get a Validation error
File upload error.
If you look in the Network tab, there is a 404 error. I tried changing APP_URL & mix_url, but that not help
![Capture](https://github.com/orchidsoftware/platform/assets/88991869/20de7978-cfe9-4309-909b-dc636e04f59b)

**Comment by tabuna** [comment, marked as answer]
The error lies in the fact that the address in the browser's address bar differs from the one specified in the settings. This is because you are using Laravel in a subdirectory, as evidenced by the presence of `/public/` in the address bar.

You can find numerous discussions, such as these:

https://github.com/laravel/framework/issues/51210#issuecomment-2078069705
https://github.com/laravel/horizon/issues/660#issuecomment-528870949
https://github.com/laravel/framework/issues/32236#issuecomment-609098208
https://github.com/laravel/telescope/issues/561#issuecomment-467585613


The framework's developers state that they do not support and do not plan to support installation in a subdirectory.

If you are an experienced user, you might attempt to resolve this using configuration files (this was previously possible for Mix, but I haven't tried it with Vite). However, such solutions do not guarantee proper functionality or support in the future.

The best option would be to set up the environment according to the framework's requirements.
